docs/components/Chips/Mobile.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { View } from "react-native";
+import { Chip } from "@jobber/components-native";
+
+export default {
+  title: "Components/Selections/Chips/Mobile",
+  component: Chip,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+    viewport: { defaultViewport: "mobile1" },
+  },
+} as ComponentMeta<typeof Chip>;
+
+const BasicTemplate: ComponentStory<typeof Chip> = args => (
+  <>
+    <View style={{ display: "flex", flexDirection: "row" }}>
+      <Chip {...args} />
+    </View>
+  </>
+);
+
+export const Basic = BasicTemplate.bind({});
+Basic.args = {
+  label: "Active chip",
+  onPress: () => alert("hi!"),
+  accessibilityLabel: "Active chip",
+  isActive: true,
+};
+
+const AccentTemplate: ComponentStory<typeof Chip> = args => (
+  <>
+    <View style={{ display: "flex", flexDirection: "row", flexWrap: "wrap" }}>
+      <Chip {...args} />
+      <Chip
+        label="Select requests"
+        accent="request"
+        icon="request"
+        onPress={() => alert("Requests")}
+        accessibilityLabel={"Select Requests"}
+        isActive={true}
+      />
+      <Chip
+        label="Select quotes"
+        accent="quote"
+        icon="quote"
+        onPress={() => alert("Quotes")}
+        accessibilityLabel={"Select Quotes"}
+        isActive={true}
+      />
+      <Chip
+        label="Select jobs"
+        accent="job"
+        icon="job"
+        onPress={() => alert("Jobs")}
+        accessibilityLabel={"Select Jobs"}
+        isActive={true}
+      />
+      <Chip
+        label="Select invoices"
+        accent="invoice"
+        icon="invoice"
+        onPress={() => alert("Invoices")}
+        accessibilityLabel={"Select Invoices"}
+        isActive={true}
+      />
+    </View>
+  </>
+);
+
+export const Accent = AccentTemplate.bind({});
+Accent.args = {
+  label: "Select clients",
+  accent: "client",
+  icon: "clients",
+  onPress: () => alert("Clients"),
+  accessibilityLabel: "Select clients",
+  isActive: true,
+};
+
+export const Dismissable = BasicTemplate.bind({});
+Dismissable.args = {
+  label: "Test Chip",
+  accent: "client",
+  icon: "clients",
+  onPress: () => alert("Clients"),
+  accessibilityLabel: "Select clients",
+  isDismissible: true,
+};
+
+export const InactiveBackgroundColor = BasicTemplate.bind({});
+InactiveBackgroundColor.args = {
+  label: "Inactive Chip",
+  accent: "client",
+  icon: "clients",
+  onPress: () => alert("Clients"),
+  accessibilityLabel: "Select clients",
+  isDismissible: true,
```

Comment:
Should an inactive chip be dismissible? If it's inactive I would assume that it should be able to be interacted with. It might be helpful if there was a use case for inactive in the docs.

Reply:
Good question, I was testing it here and the prop `inactiveBackgroundColor` just changes the color, the user still can click and interact with the Chip 🤔 

I will remove the `isDismissible` to avoid confusion, it was just a leftover from the previous example

Reply:
hm if you can still interact with it then should it be called inactive? I'm confused as to how this is meant to work.

Reply:
I'm confused too. I can raise a ticket so we can review this behavior and fix it

Reply:
Looks like it's only actually disabled when a `onPress` callback is not passed: https://github.com/GetJobber/atlantis/blob/f73435c6443b80638b1b7558dfba9a4b8f79bdd1/packages/components-native/src/Chip/Chip.tsx#L111

Reply:
Oh, ok. I think I got it. The Chip being inactive doesn't mean it's disabled. Because you can have multiple chips and select some of them to be "active", or "selected". You can still dismiss the chip if it's inactive, because it's just not select at the moment.

Reply:
oh ok sounds good.